[2025-02-23T21:06:01.009+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:05:55.859606+00:00 [queued]>
[2025-02-23T21:06:01.056+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:05:55.859606+00:00 [queued]>
[2025-02-23T21:06:01.061+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-23T21:06:01.145+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-23 21:05:55.859606+00:00
[2025-02-23T21:06:01.210+0000] {standard_task_runner.py:57} INFO - Started process 3228 to run task
[2025-02-23T21:06:01.237+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-23T21:05:55.859606+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmphq80v7eu']
[2025-02-23T21:06:01.240+0000] {standard_task_runner.py:85} INFO - Job 94: Subtask kafka_data_stream
[2025-02-23T21:06:01.626+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:05:55.859606+00:00 [running]> on host e6cb99f1bad9
[2025-02-23T21:06:02.423+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T21:05:55.859606+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T21:05:55.859606+00:00'
[2025-02-23T21:06:19.851+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-23T21:06:20.052+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-23T21:06:20.055+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-23T21:06:20.059+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-23T21:06:20.061+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-23T21:06:20.092+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250223T210555, start_date=20250223T210601, end_date=20250223T210620
[2025-02-23T21:06:20.229+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-23T21:06:20.300+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
